lab.js

```diff
@@ -52,16 +63,19 @@ function Product(params = { name: "", description: "", price: 1, brand: "", quan
     this.deleteReview = (id) => this.reviews = this.reviews.filter(r => r.ID !== id)
 
     this.deleteSize = (that) => this.sizes = this.sizes.filter(size => size !== that)
-    this.getAverageRating = () => _.meanBy(this.reviews, r => r.rating.value)
+    this.getAverageRating = (key) => _.meanBy(this.reviews, r => r.rating[key])
```

Comment:
You must return object
object = {service: avarageService ,
                 price: avaragePrice,
                 value: avarageValue,
                  quality: avarageQuality,
                  }